### DISCUSSION TITLE
Where can we find the name of event we could use?

### DISCUSSION THREAD

**Discussion by v479038280** [zauberzeug/nicegui#1615] [Q&A] [answered]
Where can we find event names such as 'dragover','dragleave','drop',that we could triger?
```
self.on('dragover.prevent', self.highlight)
self.on('dragleave', self.unhighlight)
self.on('drop', self.move_card)
self.on_drop = on_drop
```

**Comment by natankeddem** [comment]
Some of what you listed can be found here:
https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/dragover_event

**Comment by falkoschindler** [comment]
- For HTML elements you can look into the HTML reference:
https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement#events
- For Quasar-based elements you can look into the documentation of the underlying Quasar component, e.g. for QInput:
https://quasar.dev/vue-components/input (see the "Events" tab)

The NiceGUI documentation currently doesn't provide any concise list of events. It would be pretty hard to generate. But we might want to add these two references to the documentation...

**Reply by falkoschindler** [reply, marked as answer]
Did so in [6804be9](https://github.com/zauberzeug/nicegui/commit/6804be9cb5e38f3795216455f4998c946be85ef9).
